wagit/main.cpp

```diff
@@ -40,7 +47,8 @@ int main(int, char* argv[]) {
     }
     else if (command == "init")
     {
-        std::cout << CommandNotImplementedWarning(command) << std::endl;
+        Path repoPath(cmdl[2]);
+        WAGit::CreateRepository(repoPath);
```

Comment:
Why are we not storing the returned object?

Reply:
This is stateless for now. Git is stateless in general.

Reply:
Does that mean we create a new Repository object for every git command? Feels like an overkill.

Reply:
So, here's how it is.
You do a `git init`. Create a repository object and kill your machine. Next day you come in and do `git status` or something. Hence we need to keep it stateless. What a Repository object represents is an in memory state of the repo. For more complex operations like committing, this will be useful for querying. Stay tuned.